packages/web-components/src/theme/set-theme.spec.ts

```diff
@@ -66,11 +61,14 @@ test.describe('setTheme()', () => {
     await expect(div).toHaveCSS('--bar', '');
   });
 
-  test('should set and unset tokens in a light DOM subtree', async ({ page }) => {
+  test('should set and unset tokens in a light DOM subtree', async ({ fastPage, page, browserName }) => {
+    // eslint-disable-next-line playwright/no-skipped-test
+    test.skip(browserName === 'webkit', 'This test is skipped in WebKit');
```

Comment:
Is there a reason that this is skipped for WebKit? I think we need to test WebKit since because the function is supposed to be cross-browser compatible.

Reply:
I wasn't able to get these tests to pass at all on Webkit. I just pushed a change which uses `Updates.process()` to ensure that the Webkit-only reflow happens after any other queued changes have been processed.

Reply:
FYI https://github.com/microsoft/fluentui/issues/33172

Reply:
Thanks for fixing this, John!